src/main.ts

```diff
@@ -5,9 +5,17 @@ import I18NextVue from 'i18next-vue'
 import { createApp } from 'vue'
 import { createPinia } from 'pinia'
 import App from './App.vue'
+import { useMapStore } from '@/stores/map.store'
 
 import './assets/main.css'
 
+declare global {
+  interface Window {
+    vueApp: any
+    mapStore: any
+  }
+}
+
```

Comment:
This could probably be solved otherwise (especially if we chose to test the prod).

Reply:
If I understand the test files, this was meant to check the mapStore content, but I think store's content should be tested with UT but not via end to end testing.
What do you think @tkohr @mki-c2c?

Reply:
Hum, good point. However, right now we are not testing component events like clicks in the UTs, for example. To test them here, we use the `mapStore`. Not sure, if we could test them differently here, or make the `mapStore` accessible in another way, or if this should rather be tested in the UTs.

Reply:
- In UT: test click events, check if mapStore._any_content_ is updated, or check if mapStore._any_action_ have been called.

- In e2e tests: test click, check if dom is updated (and not the store content).

I think e2etesting should only manipulate the dom (checking visual updates) as it is meant to simulate the user behavior.